app/javascript/components/Ramp.jsx

```diff
@@ -73,18 +85,24 @@ const Ramp = ({
           { cdl.enabled && <div dangerouslySetInnerHTML={{ __html: cdl.destroy }}/> }
           <Tabs>
             <Tab eventKey="details" title="Details">
+            <div class='scrolling-div'>
               <MetadataDisplay showHeading={false} displayTitle={false}/>
+            </div>
             </Tab>
             { (cdl.can_stream && master_files_count != 0 && !in_progress) &&
               <Tab eventKey="transcripts" title="Transcripts">
+              <div class='scrolling-div'>
```

Comment:
In JSX `className` is used instead of `class`. This gives a warning in the console, as an invalid DOM property. So it is `className='scrolling-div'` instead of `class='scrolling-div'`

Reply:
On further thought, and after the discussion of #5390, these divs shouldn't be added. I copied them from the mockup html, but didn't add/copy any css related to that class so its superfluous. And then we don't want the metadata (and presumably section files) to scroll anyways, only the transcript which is already handled by Ramp. So I will remove these.